lib/tasks/export_permissions.rake

```diff
@@ -0,0 +1,10 @@
+require_relative '../../../foreman/db/seeds.d/020-permissions_list'
+
+desc 'Export Foreman permissions to JavaScript'
+task export_permissions: :environment do
+  formatted = PermissionsList.permissions.map { |permission| "export const #{permission[1].upcase} = '#{permission[1]}';\n" }
+  File.open('webpack/assets/javascripts/react_app/permissions.js', 'w') do |f|
```

Comment:
I'd like to use `Rails.root` as a base for a complete path in case something reuses this